tests/test_notebooks.py

```diff
@@ -1,69 +0,0 @@
-import unittest
```

Comment:
Do we test the `docs/*.ipybn` somehow?

Reply:
Yes, but
- we will not store the nb's output and not validate it as text (because its very hard to improve the js api, if we need to rerun and save all the docs manually after all small changes, and we store a lot of generated code too)
- so in make doc, we run the notebooks and generate their outputs (nbconvert with execute flag) (in this step we will get if runtime error occours)
- however the output can get worse (without error) but before releases and before bigger modifications we will check it

We discussed it with @simzer and it is maybe the best compromised solution.

Reply:
And I d like to add native js unit tests for the separated js class in the near future.

Reply:
@nyirog And we use display_javascript instead of display_html, becuse with display_javascript the js errors will be printed out in python console too not only in the browser.
(Currently uncaught in promise errors not, but it can be fixed in the js code)

Reply:
Thanks for the details.